ff_derive/Cargo.toml

```diff
@@ -12,6 +12,11 @@ license = "MIT/Apache-2.0"
 repository = "https://github.com/zkcrypto/ff"
 edition = "2018"
 
+[features]
+# enabled when generating bitvec code utilizing the version of ff's bitvec
+# ff/bits enables this
```

Comment:
Per above, we can't do this before MSRV 1.60.0.
```suggestion
```

Reply:
Handled in the push of rebased version.